eth/backend.go

```diff
@@ -884,8 +884,11 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 			  ZZZZZZZ  K   K  R   R   P       CCCC
 
 			*/
-
-			streamClient := initDataStreamClient(ctx, cfg.Zk)
+			latestForkId, err := stages.GetStageProgress(tx, stages.ForkId)
```

Comment:
Is stages.ForkId progress guaranteed to be equal to the latest downloaded batch? Is it set in the batches stage?

Reply:
yes it is at the end of stage batches